src/index.html

```diff
@@ -33,7 +33,7 @@
                 <div class="sidetab-top">
                     <img src="assets/svg/search_tab.svg" class="top-icon" />
                     <input class="sidetab" type="text" maxlength="20" placeholder="Enter stock name or symbol"
-                           ng-show="sidebarCtrl.showSearches()" ng-model="searchCtrl.query" />
+                           ng-show="sidebarCtrl.showSearches()" ng-model="searchCtrl.query" ng-keyup="searchCtrl.onSearchKeyUp($event)" />
                 </div>
```

Comment:
Keypress events don't trigger for arrow keys, rather weirdly, just down/up. I chose keyup just because it felt more analogous to a click event, where that's registered on a mousedown + up, but I don't mind switching it out